### ISSUE TITLE
I like your blog style

### ISSUE THREAD

**Issue by cizel** [ahonn/yuexun.me#1]
I like your blog style, can I make a hugo style open source and use it?

**Comment by ahonn** [comment]
Of course, you just need to include my GitHub link in the repo README file.

> I like your blog style, can I make a hugo style open source and use it?
